gcsfs/tests/test_core.py

```diff
@@ -667,15 +667,15 @@ def test_bigger_than_block_read(token_restore):
 
 @my_vcr.use_cassette(match=['all'])
 def test_current(token_restore):
-    from google.oauth2.credentials import Credentials
+    from google.auth import credentials
```

Comment:
I used a service account, not oauth.

Reply:
Is there a base class that would include both possibilities?

Reply:
(or is this already it...)

Reply:
This is it.

Reply:
It’s a little strange that they name them all Credentials.